projects/common/models/slider-element.ts

```diff
@@ -0,0 +1,31 @@
+import {FontElement, SurfaceUIElement} from '../interfaces/UIElementInterfaces';
+import { InputElement, UIElement} from './uI-element';
+import {initFontElement, initSurfaceElement} from '../util/unit-interface-initializer';
+
+export class SliderElement extends InputElement implements FontElement, SurfaceUIElement {
+  minValue: number = 0;
+  maxValue: number = 100;
+  showValues: boolean = true;
+
+  bold: boolean = false;
+  font: string = 'Roboto';
+  fontColor: string = 'black';
+  fontSize: number = 18;
+  italic: boolean = false;
+  lineHeight: number = 120;
+  underline: boolean = false;
+
+  backgroundColor: string = '#AAA0';
+
+  constructor(serializedElement: UIElement) {
+    super(serializedElement);
+    Object.assign(this, serializedElement);
+    Object.assign(this, initFontElement(serializedElement));
+    if (!serializedElement.backgroundColor) {
+      serializedElement.backgroundColor = '#d3d3d300';
+    }
+    Object.assign(this, initSurfaceElement(serializedElement));
```

Comment:
InitSurfaceElement sollte der Ordnung halber vor dem Überschreiben der defaults passieren. Wie in allen anderen Elementen.

Reply:
Nee in diesem Fall nicht. Einen ungesetzten Background setzt initSurfaceElement auf einen Wert, den ich nicht haben will. Danach kann ich nicht mehr unterscheiden, ob der Wert vom User gesetzt wurde.

Reply:
Doch serializedElement hat danach immernoch den eingelesenen Wert. Den kannst du dann immernoch benutzen. Alle anderen Elemente machen das erfolgreich so. :P